recipes/openexr/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(MSVC)
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+endif()
+
+add_executable(${PROJECT_NAME}
```

Comment:
reduce it to only the test_package file. The idea behind test package is checking your package generation, not an unit test or a complete feature exercise. Try something very simple and short enough to fit in an unique file.